Source/Data/Remotes/ParleyRemote.swift

```diff
@@ -133,7 +194,10 @@ internal class ParleyRemote {
             debugPrint("ParleyRemote.execute:: \(method) \(getUrl(path)) \(parameters ?? [:])")
             
             let request = sessionManager.request(url, method: method, parameters: parameters, encoding: JSONEncoding.default, headers: getHeaders())
-            request.validate(statusCode: 200...299).responseImage { response in
+            request
+                .validate(statusCode: 200...299)
+                .validate(parleyRemoteInterceptorProxy.validate)
+                .responseImage { response in
```

Comment:
The intereceptor adding is inconsistent. Please move the interceptors to always first do the interceptor, then validate 200...299